platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/cli/ValidateManifestFileCommand.java

```diff
@@ -45,7 +47,8 @@ private void setDir(final Path dir) {
 
     @Override
     public @NonNull Integer call() throws IOException {
-        EmergencyRecoveryFile.read(dir, true);
+        final StateConfig stateConfig = ConfigurationHolder.getConfigData(StateConfig.class);
```

Comment:
This is an added static call to the `ConfigurationHolder` for the `StateConfig`.  My impression of this PR was to remove these static calls instead of adding them.

Reply:
You’ve got me :) I had actually no idea how to access the configuration from a Command so I decided to use the `ConfigurationHolder` here but remove the call inside.

Reply:
Found a solution :)